pyro/distributions/torch_patch.py

```diff
@@ -2,7 +2,9 @@
 
 import torch
 
-assert torch.__version__.startswith('1.')
+if 'READTHEDOCS' not in os.environ:
```

Comment:
import os

Reply:
thanks. you can see the the thorough testing this went through